config/nginx-default.conf

```diff
@@ -1,6 +1,7 @@
 server {
     listen      80;
     client_max_body_size 50m;
+    proxy_read_timeout 120;
```

Comment:
it's usually written as `120s`, is it the same?

Reply:
I didn't take the example from the source. I added the s.